src/test/java/it/pagopa/pn/bff/pnclient/externalregistries/PnExternalRegistriesClientImpTest.java

```diff
@@ -0,0 +1,118 @@
+package it.pagopa.pn.bff.pnclient.externalregistries;
+
+import it.pagopa.pn.bff.exceptions.PnBffException;
+import it.pagopa.pn.bff.generated.openapi.msclient.external_registries.api.InfoPaApi;
+import it.pagopa.pn.bff.generated.openapi.msclient.external_registries.model.CxTypeAuthFleet;
+import it.pagopa.pn.bff.generated.openapi.msclient.external_registries.model.InstitutionResourcePN;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.web.reactive.function.client.WebClientResponseException;
+import reactor.core.publisher.Flux;
+import reactor.test.StepVerifier;
+
+import java.util.List;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@ContextConfiguration(classes = {PnExternalRegistriesClientImpl.class})
+@ExtendWith(SpringExtension.class)
+public class PnExternalRegistriesClientImpTest {
+    @Autowired
+    private PnExternalRegistriesClientImpl pnExternalRegistriesClientImpl;
+    @MockBean
+    private InfoPaApi infoPaApi;
+
+    @Test
+    void getInstitutionsTest() {
+        when(infoPaApi.getInstitutions(
+                Mockito.<String>any(),
```

Comment:
Use Mockito.anyString, Mockito.anyBoolean, Mockito.any(CxTypeAuthFleet.class) and so on. Do it for all tests